plug.py

```diff
@@ -98,7 +98,7 @@ def notify_system_changed(timestamp, system):
         system_changed = _get_plugin_func(plugname, "system_changed")
         if system_changed:
             try:
-                system_changed(timestamp, system)
+                system_changed(timestamp, system, coordinates)
```

Comment:
Use co_argcount to detect old plugins.
Add a **kwargs parameter instead of an explicit coordinates parameter to avoid this problem in future.